packages/ndla-ui/src/LicenseByline/LicenseLink.tsx

```diff
@@ -25,16 +26,22 @@ const StyledSafeLink = styled(SafeLink, {
     _focusWithin: {
       textDecoration: "none",
     },
+    mobileWideDown: {
+      _disabled: {
```

Comment:
fant ikke `_hidden` eller lignende som passet bedre. åpen for forslag

Reply:
Du kan sette `hidden={true}` på komponenten, så fjernes den automatisk.

Reply:
funker litt dårlig når jeg skal bare ha det på `mobileWideDown`